src/page-multi-facility/ScenarioSidebar.tsx

```diff
@@ -112,21 +117,30 @@ const ScenarioSidebar: React.FC<Props> = (props) => {
   return (
     <div className="flex flex-col w-1/4 mr-24">
       <div className="flex-1 flex flex-col pb-4">
-        {showScenarioLibrary && (
-          <ScenarioLibraryModal
-            trigger={<IconFolder alt="folder" src={iconFolderSrc} />}
+        <ScenarioName>
+          {showScenarioLibrary && (
+            <ScenarioLibraryModal
+              trigger={
+                <IconFolder
+                  style={{
+                    marginTop: "8px",
+                  }}
```

Comment:
A side effect of embedding these trigger components within the modal component itself is that we have to do any custom styling inline.  I think that's ok.